lib/gui/app/components/modal/services/modal.js

```diff
@@ -44,7 +45,7 @@ module.exports = function ($uibModal, $q) {
     })
 
     analytics.logEvent('Open modal', {
-      template: options.template
+      name: options.name
```

Comment:
Should we do the same for "close" events?

Reply:
Yeah that's a good idea, done now